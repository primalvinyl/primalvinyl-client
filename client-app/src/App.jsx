import React from 'react';
import { BreakpointProvider } from './utilities/BreakpointProvider';
import logo from './images/logo.svg';
import './App.scss';

function App() {
    return (
        <div className="App">
            <BreakpointProvider>
                <header className="App-header">
                    <img src={logo} className="App-logo" alt="logo" />
                    <p>Edit <code>src/App.js</code> and save to reload.</p>
                </header>
            </BreakpointProvider>
        </div>
    );
}

export default App;
