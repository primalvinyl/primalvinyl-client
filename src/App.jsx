import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { BreakpointProvider } from './utilities/BreakpointProvider';
import Routes from './routes/Routes';

const App = () => (
    <BrowserRouter>
        <BreakpointProvider>
            <div className="app">
                <Routes />
            </div>
        </BreakpointProvider>
    </BrowserRouter>
);

export default App;
