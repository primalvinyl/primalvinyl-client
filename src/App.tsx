import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { BreakpointProvider } from './components/presentation/BreakpointProvider';
import Routes from './routes/Routes';

const App = (): React.ReactElement => (
    <BrowserRouter>
        <BreakpointProvider>
            <div className="app">
                <Routes />
            </div>
        </BreakpointProvider>
    </BrowserRouter>
);

export default App;
