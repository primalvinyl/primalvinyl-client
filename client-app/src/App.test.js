import React from 'react';
import { createStore } from 'redux';
import { Provider } from 'react-redux';
import { shallow } from 'enzyme';
import Renderer from 'react-test-renderer';
import rootReducer from './reducers';
import App from './App'; 

describe('App', () => {
    it('renders', () => {
        const store = createStore(rootReducer, {});
        const myWrapper = shallow(
            <Provider store={store}>
                <App />
            </Provider>
        );
        expect(myWrapper.exists());
    });
    it('matches snapshot', () => {
        const component = Renderer.create(<App />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
