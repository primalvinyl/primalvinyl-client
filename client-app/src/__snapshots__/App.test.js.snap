// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<div
  className="app"
>
  <div>
    <header>
      <div
        className="wrapper"
      >
        <h1>
          Header
        </h1>
      </div>
    </header>
    <main>
      <div
        className="home-page"
      >
        <div
          className="wrapper"
        >
          <h2>
            Home Page
          </h2>
        </div>
      </div>
    </main>
  </div>
  <footer>
    <div
      className="wrapper"
    >
      Footer
    </div>
  </footer>
</div>
`;
